slsim/Sources/supernovae.py

```diff
@@ -101,8 +102,25 @@ def get_apparent_magnitude(self, time, band, zpsys="AB"):
 
         :return: magnitude of source
         """
+        minphase = self.source.minphase()
+        """
+        mag_at_minphase = self.bandmag(band, zpsys, minphase)
+        time = np.atleast_1d(time)
+        for i in range(np.size(time)):
+            if time[i] < minphase - 20:
+                time[i] = minphase - 20
 
-        return self.bandmag(band, zpsys, time)
+        x = np.array([time[0], minphase])
```

Comment:
I don't really see where these lines of code are being used further down

Reply:
Sorry, I forgot to delete those lines when I was testing stuff. Fixed now